docs/changes/extern/12716.bugfix.rst

```diff
@@ -0,0 +1,5 @@
+Local CSS file `jquery.dataTables.css` refers to several images called
+`sort*.png`. They are the little arrows next to each sortable column. These
+images exist at the CDN (https://cdn.datatables.net/1.10.12/images/), but they
+were missing in this repo. Images required to display the corresponding arrows
+were added.
```

Comment:
The bugfix entry should focus on the user perspective, not developer details. I'd suggest below. BTW does this apply to `show_in_notebook` as well?

```suggestion
Fixed a bug in ``Table.show_in_browser`` using the ``jsviewer=True`` option
to display the table with sortable columns. Previously the sort direction arrows
were not being shown due to missing image files for the arrows.
```

The useful information originally in the change log should probably go in the PR description.

Reply:
>. BTW does this apply to `show_in_notebook` as well?

I don't think so. `show_in_notebook` uses the `JSViewer` class with the default `use_local_files=False`. Local files are used when `use_local_files=True`, like  in `show_in_browser`.